packages/networks/bin/utils/contracts.ts

```diff
@@ -44,14 +44,24 @@ export const getAllAddresses = async ({ network }) => {
     addresses[`UnlockOwner`] = unlockDaoToken.address
   }
 
-  // TODO: get all hooks
-  if (hooks) {
-    if (hooks.onKeyPurchaseHook) {
-      hooks.onKeyPurchaseHook.map(({ address, id }) => {
-        addresses[`onKeyPurchaseHook_${id}`] = address
-      })
-    }
-  }
+  // TODO: get hooks from type file
+  const requiredHooks = [
```

Comment:
Do you think you could do that in this PR so this does not slip thru the cracks?

Reply:
yes sure